app/uk/gov/hmrc/gform/controllers/AuthenticatedRequestActions.scala

```diff
@@ -121,10 +121,19 @@ class AuthenticatedRequestActions(
     f: Request[AnyContent] => LangADT => AuthCacheWithoutForm => Future[Result]): Action[AnyContent] = Action.async {
     implicit request =>
       implicit val l: LangADT = getCurrentLanguage(request)
+
+      val assumedIdentity = getAssumedIdentityCookie(request.cookies.get("caseworker-assumed-identity"))
```

Comment:
Lines 125 and 126 are duplication of lines 167 and 168. Surely there must be a way how remove this duplication.

Reply:
I will re-work it to pass the cookie straight into the function and handle the logic to deal with it inside the ALB AUTH function.